--- src/routes/projectview/ProjectView.svelte
+++ src/routes/projectview/ProjectView.svelte
@@ -1089,7 +1089,7 @@
				bind:isProjectSettingsModalOpen={isProjectSettingsModalOpen}
				bind:isTranscriptionSettingsModalOpen={isTranscriptionSettingsModalOpen}
				bind:isTranslationSettingsModalOpen={isTranslationSettingsModalOpen}
-				tableViewRef={dataViewRef?.tableViewRef}
+				dataViewRef={dataViewRef}
				{hasHeaders}
				{isTable}
                 on:requestImageExport={() => dataViewRef?.triggerImageExport()}
--- src/lib/components/projectview/data/DataTopBar.svelte
+++ src/lib/components/projectview/data/DataTopBar.svelte
@@ -25,7 +25,7 @@

     export let hasHeaders = true;
     export let isTable = false;
-    export let tableViewRef = null; // Passed down from ProjectView -> DataView to here
+    export let dataViewRef = null; // Passed down from ProjectView

     // Modals
     let showTableExportModal = false;
@@ -807,7 +807,7 @@
 <TableExportModal
     bind:showModal={showTableExportModal}
     tablePath={isTable ? $project.selectedDocumentPath : null}
-    getExportData={() => { if (isTable && tableViewRef) { return tableViewRef.getExportData(); } else { return null; } }}
+    getExportData={() => { if (isTable && dataViewRef) { return dataViewRef.getExportData(); } else { return null; } }}
     on:confirm={() => message('Table exported successfully.', { title: 'Success', type: 'info' })}
     on:close={() => showTableExportModal = false}
 />
